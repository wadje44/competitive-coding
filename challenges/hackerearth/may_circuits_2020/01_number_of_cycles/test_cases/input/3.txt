5
1000
5556
562
889
12