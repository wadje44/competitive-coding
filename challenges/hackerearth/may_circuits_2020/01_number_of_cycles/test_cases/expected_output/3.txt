999001
30863581
315283
789433
133